### ISSUE TITLE
pip install flashtext fails with following error when python-sphinx is missing.

### ISSUE THREAD

**Issue by vi3k6i5** [vi3k6i5/flashtext#4]
```
pip install flashtext==1.7
Collecting flashtext==1.7
  Downloading flashtext-1.7.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/yh/8xf7wskn0d3fdfwnrj1vgh_r0000gn/T/pip-build-pdaQyo/flashtext/setup.py", line 2, in <module>
        from sphinx.setup_command import BuildDoc
    ImportError: No module named sphinx.setup_command
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /private/var/folders/yh/8xf7wskn0d3fdfwnrj1vgh_r0000gn/T/pip-build-pdaQyo/flashtext/
```
Resolution: ```sudo apt-get install python-sphinx```

**Comment by vi3k6i5** [comment]
Asked a question on Stack overflow for the same:

https://stackoverflow.com/questions/46111049/how-to-make-builddoc-work-only-when-python-sphinx-is-installed-from-setup-py-fil/46112100#46112100

Got suggestion. Implementing that in v1.8

**Comment by vi3k6i5** [comment]
Fixed in v1.8
